frontend/util.lua

```diff
@@ -563,6 +563,52 @@ function util.makePath(path)
     return lfs.mkdir(path)
 end
 
+--- As `rm`
+-- @string path of the file to remove
+-- @treturn bool true on success; nil, err_message on error
+function util.removeFile(file)
+    local lfs = require("libs/libkoreader-lfs")
+    if file and lfs.attributes(file, "mode") == "file" then
+        return os.remove(file)
+    elseif file then
+        return nil, file .. " is not a file"
+    else
+        return nil, "file is nil"
+    end
+end
+
+-- Gets total, used and available bytes for a given directory.
+-- @string path
+-- @treturn table with total, used and available bytes
+function util.diskUsage(dir)
+    -- safe way of testing df & awk
+    local function doCommand(d)
+        local handle = io.popen("df -k " .. d .. " 2>&1 | awk '$3 ~ /[0-9]+/ { print $2,$3,$4 }' 2>&1 || echo ::ERROR::")
+        if not handle then return end
+        local output = handle:read("*all")
```

Comment:
I don't know if it's stictly needed or if Lua will properly close the handle when GC'ing.
But we usually follow that with `handle:close()`

Reply:
From https://www.lua.org/work/doc/manual.html

> Note that files are automatically closed when their handles are garbage collected, but that takes an unpredictable amount of time to happen.

I would expect the handle to be able to be collected at the end of the function, but the quote doesn't say that. So better stick with `handle:close()`

Reply:
Not strictly, but then it won't close until whenever it decides to garbage collect.